Source/Core/Core/HW/WiimoteEmu/Dynamics.cpp

```diff
@@ -220,7 +222,10 @@ void Wiimote::GetAccelData(NormalizedAccelData* accel)
   const bool is_sideways = IsSideways();
   const bool is_upright = IsUpright();
 
-  EmulateTilt(accel, m_tilt, is_sideways, is_upright);
+  // Semi-hacky: Add cursor Y position into the pitch to fix Skyward Sword's pointer.
+  const auto cursor_tilt = m_ir->GetState(true).y * MathUtil::PI * -0.25;
```

Comment:
Wheres that `-0.25` from, and is it important enough to be documented/constified?

Reply:
It's a 1/4 of PI radians (45 degrees). This is a temporary a hack but I'll adjust the comment.